src/server/config/schema.js

```diff
@@ -27,6 +27,21 @@ module.exports = () => Joi.object({
     exclusive: Joi.boolean().default(false)
   }).default(),
 
+  cpu: Joi.object({
+    cgroup: Joi.object({
+      path: Joi.object({
```

Comment:
The name was discussed [here](https://github.com/elastic/logstash/issues/6797#issuecomment-286605180) for Logstash/ES. While I agree we could omit "override" it maintains consistency with the other projects.